public/data/AlimurtuzaCodes.json

```diff
@@ -0,0 +1,11 @@
+{
+ "name": "Alimurtuza",
+ "bio": "Mobile and Web App Developer",
+ "avatar": "https://github.com/AlimurtuzaCodes.png",
+ "portfolio": "https://github.com/AlimurtuzaCodes/",
+ "skills": ["React", "React Native", "Javascript", "Java", "C++"],
+ "social": {
+   "GitHub": "https://github.com/AlimurtuzaCodes",
+   "LinkedIn": "https://www.linkedin.com/in/p-alimurtuza"
+ }
+},
```

Comment:
Remove the last `, ` comma from line number 11.

Additionally you can also add your location on profile.

Reply:
Done.